### DISCUSSION TITLE
Make a flavour that excludes this library (fdroid)

### DISCUSSION THREAD

**Discussion by ondrejnedoma** [invertase/react-native-firebase#7588] [Q&A]
I use Firebase only for notifications, which can be implemented in a different way, and the fdroid store does not allow Firebase in the apps on their store. So I want to be able to build the app without the Firebase dependency. There can't be any mentions of Firebase in the final apk file. How would I approach this?

**Comment by mikehardy** [comment]
Sorry, that's not something we can answer for you.
I'm a fan of f-droid and a fan of open source (I maintain AnkiDroid with a few million installs, and we distribute by f-droid as one of our channels!)
But there are many many ways to go about this and I can't prescribe one as they all go down to native android gradle work and that's out of scope for this repo

You will probably need to do some sort of gradle post work where you trim out dependencies though
